src/Snackbar/Snackbar.js

```diff
@@ -0,0 +1,247 @@
+// @flow weak
+
+import React, { PropTypes, Component } from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import customPropTypes from 'material-ui/utils/customPropTypes';
+import SnackbarContent from './SnackbarContent';
+import ClickAwayListener from '../internal/ClickAwayListener';
+
+
+export const styleSheet = createStyleSheet('MuiSnackbar', (theme) => {
+  const { zIndex, transitions, breakpoints } = theme;
+
+  const transitionString = `${transitions.create('transform', {
+    duration: 400,
+    easing: transitions.easing.easeInOut,
+    delay: 0,
+  })}, ${transitions.create('visibility', {
+    duration: 400,
+    easing: transitions.easing.easeInOut,
+    delay: 0,
+  })}`;
+  return {
+    root: {
+      position: 'fixed',
+      left: '50%',
+      display: 'flex',
+      bottom: 0,
+      zIndex: zIndex.snackbar,
+      transition: transitionString,
+    },
+    [breakpoints.down('sm')]: {
```

Comment:
We avoid using `down` as it's not in the mobile-first design principle (it's never used actually).
Here, you don't need it.